cupy/linalg/norms.py

```diff
@@ -57,7 +54,7 @@ def norm(x, ord=None, axis=None, keepdims=False):
             return abs(x).min(axis=axis, keepdims=keepdims)
         elif ord == 0:
             # Zero norm
-            return (x != 0).sum(axis=axis, keepdims=keepdims, dtype=x.dtype)
+            return (x != 0).sum(axis=axis, keepdims=keepdims, dtype='d')
```

Comment:
Yes, NumPy does.
I don't know reason of using this type. But, I think that such a case is normal.
There is a possibility that it will change. 
Do you need comment?